### ISSUE TITLE
Increase font size 

### ISSUE THREAD

**Issue by hsarji** [holzschu/Carnets#151]
Is there a way to increase the font size of rendered Markdown? 

My research shows that I should be able to manipulate a css file, but I can't get to it from my iPad. The app's data seems to be at `/private/var/mobile/Containers/Data/Application/0E364424-6922-4AA3-A10D-68CAD881D4EB`. 

I discovered a workaround, but I need help with it. I can run this code:

```
%%html
<style type='text/css'>
.CodeMirror{
font-size: 22px;
</style>
```

The code increases the font of cells when I'm editing them but not after I run them (which makes it hard to see Markdown after I run it). Which CSS values would increase the font of Markdown after I run the cell? I've been experimenting with different values, but I have yet to figure it out. See <https://gist.github.com/ohgyun/986a5c1718e49f8d6da6d0cb5e79438e>

This workaround isn't ideal because the code gets exported when I export the notebook to a Markdown file and I have to clean up my text after exporting it.

**Comment by holzschu** [comment]
Hi, this directory, which corresponds to `~` or `$HOME` cannot be accessed by the user (it has no read or write permission). `~/Documents/` is the directory where most things are stored. You can place customization files in `~/Documents/.jupyter/custom/`, for example `~/Documents/.jupyter/custom/custom.css`. 
See for example here: https://byte7.github.io/blog/Cool-Jupyter/

Basically, for all documentation that says `~/.something`, you should be able to make it work with Carnets with `~/Documents/.something`.

**Comment by hsarji** [comment]
Thank you for clarifying. I added `custom.css` to `iCloud Drive/Carnets/Documents/.jupyter/custom/custom.css` but I haven't been able to get the CSS to change anything. Am I saving the file in the right place? 

To follow up on my workaround, I got the following to increase the size of all text. (Sadly, it also increases the size of the Run button and the navigation bar.)

```
%%html
<style type='text/css'>
*{ 
  font-size: 18px;
  line-height: 26px;
}
</style>
```

I also can eliminate editing the Markdown that I export by adding a Markdown cell before the code with `<!--` and another Markdown cell below the code with `-->`.

**Comment by holzschu** [comment]
You did everything right, except that the configuration files go in `on my iPad/Documents/.jupyter/custom/custom.css`, which is the same as `~/Documents/.jupyter/custom/custom.css`.

**Comment by hsarji** [comment]
I placed the configuration files where you showed and they work. Thank you! Two related questions: (1) When I attach an image to a cell, where does Carnets save that image? (2) By any chance, do you know how I can modify the CSS from my last post so the size of the Run button doesn't get larger?

**Comment by hsarji** [comment]
Regarding my related question (1), I can answer my own question. I created a directory that mirrors the one on my Jekyll blog, `assets/img`. I dragged the images from the Apple Photos app into the `img` folder using the Files app. (This converted the photo to jpeg format.) And then I was able to use this Markdown syntax in Carnets: `![title](assets/img/python1.jpeg)`. The upside is that I can simply export the Markdown and the syntax will work in my Jekyll blog. The downside is that image doesn't export automatically. I have to move the image to the corresponding directory in the Working Copy app. 
`

**Comment by holzschu** [comment]
The answer to question 1 is, I think, "in the current directory". But there's a catch: on iOS, you don't always have read-write permission on the current directory (you have automatic permission in `on my iPad/Carnets` and `iCloud Drive/Carnets`). You can request permission on a folder for Carnets by clicking on the folder icon at the right of the toolbar, then navigating to the folder. After that, files created inside Carnets are created also inside the folder.

To answer question 2, first method: 
- `! cp ~/Documents/.jupyter/custom/custom.css ~/Documents`
- edit the file `custom.css` using your favorite editor app
- `! cp ~/Documents/custom.css ~/Documents/.jupyter/custom/custom.css`

Second method: 
- install a-Shell or a-Shell mini (different apps, same developer) 
- inside a-Shell, `pickFolder`, navigate to the Carnets directory
- type `vim .jupyter/custom/custom.css`
The second method is recommended only if you know enough about vim (the editor) to edit a file and save it.

**Comment by hsarji** [comment]
Regarding question 2, I was able to edit the custom.css using Textastic and it is working well. I was just wondering if you knew how to tweak the CSS to solve a specific issue, but it's not a big issue and I can continue playing with the CSS. So, with your help, I'm now able to modify the default font in Carnets. Thank you!

Regarding question 1, I realized that going from Jupyter to Jekyll isn't going to be as easy as I first thought. Jupyter uses the following Markdown to link to an image: `![title](assets/img/python1.jpeg)`. Jekyll uses the following Markdown: `![title](/assets/img/python1.jpeg)`. That `/` throws things off!
